runtime.md

```diff
@@ -1,10 +1,5 @@
 # Runtime and Lifecycle
 
-## Scope of a Container
-
-Barring access control concerns, the entity using a runtime to create a container MUST be able to use the operations defined in this specification against that same container.
-Whether other entities using the same, or other, instance of the runtime can see that container is out of scope of this specification.
-
```

Comment:
not sure why you removed this.  I thought you were asking for text related to the scope of a runtime's data.